augur/application/schema/alembic/versions/2022-09-28_12_add_user_repo_table.py

```diff
@@ -0,0 +1,54 @@
+"""Add user repo table
+
+Revision ID: 12
```

Comment:
@ABrain7710 I need to change the revision id on this file

Reply:
@ABrain7710 : I fixed this.